### DISCUSSION TITLE
Why would apps open when running them from "drive_c" but not the main interface?

### DISCUSSION THREAD

**Discussion by dsifriend** [Whisky-App/Whisky#306] [Q&A]
This might be worth its own ticket, but I'll ask here first in case I'm missing something obvious and it's not a bug: What is different about running an app from the "Run…" dialog and from the "Programs" list/shortcuts that would make it so some programs would run correctly with the former but crash immediately through the latter?

I've been having trouble getting Steam games to run properly (Baldur's Gate 3, in particular) and even the Steam App can be finnicky _unless I find them in the bottle's 'drive_c' to run from there_. Is it an issue with my config? How can I get consistent results from both methods?

**Comment by Jayian1890** [comment]
I just recently made a bug report(https://github.com/IsaacMarovitz/Whisky/issues/304) on this exact same issue. From what I've seen. It's possibly due to the "/unix" flag not being set for apps ran from the Programs list(they're executed directly), but is being set for apps being ran via the "run..." dialog. I'm not sure what that flag does. But it does appear that not having it causes a number of games to simply not run. For me personally, none of my titles run unless it's set. Northgard, Octopath 2, and Bannerlord just to name a few.

I would upload my build, but I'm not sure if that's allowed or not. I'm about to submit a pull request for the developer to review. Perhaps it's a fix, perhaps it's a bandaid. kind of depends on what the dev's intentions are with there being two different methods of execution.

**Reply by Jayian1890** [reply]
Was looking through previous pull requests, and came across this one in particular https://github.com/IsaacMarovitz/Whisky/commit/08acc0bff1595af17f63d4b72d9b8f525b48daf4

it appears the "/unix" flag was intentionally(or erroneously) removed on June 21.

**Reply by dsifriend** [reply]
That would do it. I'd make that pull request, at least to get some clarity on the issue, if not to fix it.

**Reply by Jayian1890** [reply]
Pull Request: https://github.com/IsaacMarovitz/Whisky/pull/311

now we wait to see what the developer thinks.

**Reply by alver1584** [reply]
@Jayian1890  are you saying you can play m&b 2 Bannerlord through whiskey??
